### ISSUE TITLE
the email sent is not localised

### ISSUE THREAD

**Issue by supermario12342003** [payloadcms/plugin-form-builder#69]
Hi,

It seems that it always send the email in the main language. Even though i have filled the email content in both language.

Is this a bug, or there is a setting in config i need to change?

**Comment by denolfe** [comment]
If you're referring to the reset password and verify emails, these are not localized currently.

However, Payload does provide a way to customize these. You can see how this is done via this example: https://github.com/payloadcms/payload/tree/main/examples/email

**Comment by supermario12342003** [comment]
no, im not refering to the reset password or the verify emails. 

Im talking bout the email feature of this plugin of this repo: [plugin-form-builder](https://github.com/payloadcms/plugin-form-builder).

<img width="1440" alt="Screenshot 2023-11-24 at 22 03 06" src="https://github.com/payloadcms/plugin-form-builder/assets/6221160/a9ce07da-b24c-4e80-9fd4-c2bd57910a75">

**Comment by jacobsfletch** [comment]
This plugin is now being maintained in the [Packages Directory](https://github.com/payloadcms/payload/tree/main/packages/plugin-form-builder) of the [Payload Monorepo](https://github.com/payloadcms/payload). This repo will soon be archived and all open issues be closed. This issue, however, will be transferred over. Please refer to [this open discussion](https://github.com/payloadcms/payload/discussions/4466) for more details.
